ideas/forms.py

```diff
@@ -29,9 +32,11 @@ class Meta:
 class NonAnonymousIdeaCreateForm(autocomplete.FutureModelForm, ModelForm):
     """Form for authenticated users"""
 
+    captcha = ReCaptchaField() 
+
     class Meta:
         model = Idea
-        fields = ['title', 'concept', 'tags', 'visibility']
+        fields = ['title', 'concept', 'tags', 'visibility', 'captcha']
```

Comment:
If you could adapt the test cases for this change, it would help pass this pull-request